task_5.py

```diff
@@ -0,0 +1,33 @@
+class TestCase:
+
+    def __init__(self):
+        self.steps = {}
+        self.result = None
+
+    def set_step(self, step_number, step_text):
+        self.steps[step_number] = step_text
+
+    def delete_step(self, step_number):
+        del self.steps[step_number]
```

Comment:
Тут лучше делать проверку на наличие элемента

Reply:
добавил условие, что удаляемый ключ существует в словаре

Reply:
добавил условие, что удаляемый ключ существует в словаре
```suggestion
        if step_number in self.steps:
            del self.steps[step_number]
```